src/theme/widths.ts

```diff
@@ -0,0 +1,7 @@
+import { Widths } from './widths.types';
+
+const widths: Widths = {
```

Comment:
I'd rather rename this to `grid/layout.ts` as it'd be more generic - usable for `reportWidth, layoutWidth, gutter` etc